Formula/llvm.rb

```diff
@@ -209,11 +266,11 @@ def install
     if build.with? "clang"
       (share/"clang/tools").install Dir["tools/clang/tools/scan-{build,view}"]
       if build.head?
-        inreplace "#{share}/clang/tools/scan-build/bin/scan-build", "$RealBin/bin/clang", "#{bin}/clang"
+        # inreplace "#{share}/clang/tools/scan-build/bin/scan-build", "$RealBin/bin/clang", "#{bin}/clang"
```

Comment:
I don't know why these exist, so I commented them out. `$RealBin/bin/clang` string does exist in `scan-build` script but I do see that the script checks whether `clang` exists. So, I commented them out for now before I know for sure if they are necessary.